tests/specs/3way/lobby.spec.ts

```diff
@@ -5,9 +5,13 @@ import type PreMeetingScreen from '../../pageobjects/PreMeetingScreen';
 
 describe('Lobby', () => {
     it('joining the meeting', async () => {
+        await hangupAllParticipants();
```

Comment:
Same here, do we need hangup all when the tests are starting? No participant has joined?